incuna-sass/mixins/_font-face.sass

```diff
@@ -73,10 +60,7 @@
         // If neither IE8 or woff2 support is required output this:
         @else
             src: font-url($filepath + ".woff") format('woff'), font-url($filepath + ".otf") format('opentype')
-
-// Use this mixin to set fonts in your project
-@mixin font($style-name, $category: sans-serif)
-    font:
-        family: '#{$style-name}', $category
-        style: normal
-        weight: normal
+        // Weight and style must be below the src declaration or will not work
+        font:
```

Comment:
sadly yes, the only way it works is if you set the weight and style after the `src` and `font-family` has to come first, slightly awkward, which is probably why we didn't figure it out until now